README.md

```diff
@@ -515,6 +517,16 @@ A `service_discovery` block configures [Amazon Service Discovery](https://docs.a
 
 * `namespace_id` – (Required) The ID of the namespace to use for DNS configuration.
 
+* `container_name` - (Required) The name of the container to use
+with the service discovery service as it appears in the container
+definition (by default, `containers.json`). Defaults to the value
+specified in the [Load Balancer block](#load_balancer).
+
+* `container_port` - (Required) The port on the container to use
```

Comment:
The service discovery map does not support container_port as input.

Reply:
Removed.